scripts/setup_db.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+set -e
+
+name="manga"
+port=5436
+
+docker container inspect $name >/dev/null 2>&1 || docker run -d --restart unless-stopped --name $name -p $port:5432 -e POSTGRES_PASSWORD=password postgres:latest >/dev/null
+
+echo 'export DATABASE_URL="postgresql://postgres:password@localhost:'$port'/postgres"' >>.env
```

Comment:
I see this line as problematic bc `>>` means it appends so it could export the same var multiple times but only having `>` would override any manual configuration

Reply:
I'd see the usage of > problematic, since it'll overwrite the file as a whole

Reply:
yeah I agree, both methods are flawed as to why I am not quite content with the approach itself

Reply:
Could just print it out instead

Reply:
But is exporting the same var multiple times problematic? If so, why?